spyder/plugins/editor/widgets/tests/conftest.py

```diff
@@ -126,6 +126,37 @@ def teardown():
     return completions
 
 
+@pytest.fixture
+def mock_codeeditor(qtbot_module, request):
+    """CodeEditor instance with ability to mock the completions response"""
+    # Create a CodeEditor instance
+    editor = codeeditor_factory()
+    qtbot_module.addWidget(editor)
+    editor.show()
+
+    mock = Mock()
+    mock.side_effect = lambda *args: None
```

Comment:
Please add a comment above this line saying that `mock.side_effect` needs to be redefined in tests that depend on this fixture